### ISSUE TITLE
Can't download dataset: [Errno 13] Permission denied

### ISSUE THREAD

**Issue by adsnash** [fMoW/dataset#3]
Is this dataset still available? I'm trying to download it with `aws s3 sync s3://fmow-full . --request-payer requester` and I'm getting `[Errno 13] Permission denied`

**Comment by mukhery** [comment]
Both fMoW-full and fMoW-rgb datasets are available for free (i.e., without requester pays) at s3://spacenet-dataset/fmow/. I recommend downloading the data from there.

Regarding the s3://fmow-full and s3://fmow-rgb buckets, I'm looking into what happened to them now. We were planning to eventually discontinue those buckets now that the data is available at s3://spacenet-dataset/fmow.

**Comment by adsnash** [comment]
Ok great, I think I also had some permission errors on my end as well. Thanks fir the help!

**Comment by VinniaKemala** [comment]
Hi @mukhery , 
I was trying to download it with `aws s3 ls s3://spacenet-dataset/fmow/fmow-full/ `and `aws s3 ls s3://spacenet-dataset/fmow/fmow-rgb/` but nothing happened. 
I also got this error 
`fatal error: An error occurred (404) when calling the HeadObject operation: Key "fmow/fmow-full/manifest.json.bz2" does not exist` when I tried to download manifest.json.bz2 files. 
Is this dataset still available?
